packages/expo-module-template/package.json

```diff
@@ -1,20 +1,35 @@
 {
   "name": "expo-module-template",
   "version": "2.0.0-rc.1",
-  "description": "Expo Module Template standalone module",
-  "main": "index.js",
+  "description": "ExpoModuleTemplate standalone module",
+  "main": "build/index.js",
+  "types": "build/index.d.ts",
+  "scripts": {
+    "build": "expo-module build",
+    "clean": "expo-module clean",
+    "test": "expo-module test",
+    "postinstall": "expo-module postinstall",
```

Comment:
Also, we've recently removed `postinstall` script from modules. See https://github.com/expo/expo/commit/0b87c1dfad145bac64ee0decd40a3e54d91cbc75